src/ui/ui.css

```diff
@@ -19,6 +19,36 @@ body {
   margin: 0;
 }
 
+/**
+ * Since the desktop view has the parent iframe (.swg-dialog) transperant,
+ * this style adds the background with borders to the loading indicator.
+ */
+@media (min-width: 630px) {
+  .swg-loading-container {
+    width: 550px !important;
+    height: 148px !important;
```

Comment:
Hmm. What happens in the transition states? E.g. when offers view transits to the pay confirmation? It'd seem like this will be broken with loading container laid out over/under size and double shadow. No?

Reply:
I have tested this few times, and transition seems working fine.